### DISCUSSION TITLE
Enabling code highlighting

### DISCUSSION THREAD

**Discussion by Altho** [ssssota/svelte-exmarkdown#32] [Q&A] [answered]
Hello, is there any plugins for allowing code highlighting ? I can try to make one following the example repo's example but I was wondering if one was around already

Thanks for your work !

**Comment by ssssota** [comment, marked as answer]
Thank you for asking!

I don't have a plugin for this.
I recommend using rehype-highlight.

e.g. https://stackblitz.com/edit/svelte-exmarkdown-code-highlight?file=src%2Fhighlight-plugin.ts

(I know that's not a good example... 😫)

**Reply by Altho** [reply]
Awesome, thank you so much !
